### DISCUSSION TITLE
7.10.0 setValue doesn't behave as expected when value is Object

### DISCUSSION THREAD

**Discussion by Seke1412** [react-hook-form/react-hook-form#5827] [Q&A] [answered]
codesandbox: https://codesandbox.io/s/react-hook-form-v7-controller-forked-cppf2?file=/src/index.js

Version 7.9.0 and before

![Screen Shot 2021-07-03 at 7 26 21 PM](https://user-images.githubusercontent.com/3940350/124355926-3a5e1d80-dc46-11eb-88a2-08e3aaf8ff5d.png)

Version 7.10.0 
![Screen Shot 2021-07-03 at 7 26 30 PM](https://user-images.githubusercontent.com/3940350/124355949-4a75fd00-dc46-11eb-9726-8219f59228fd.png)

**Comment by bluebill1049** [comment, marked as answer]
provide a codesandbox plz

**Reply by bluebill1049** [reply]
fixed on the latest patch 👍

**Reply by bluebill1049** [reply]
<img width="593" alt="Screen Shot 2021-07-05 at 2 14 12 pm" src="https://user-images.githubusercontent.com/10513364/124416867-47405580-dd9b-11eb-9e49-90b376d7fbd4.png">

**Reply by Seke1412** [reply]
love you 3000 @bluebill1049

**Reply by bluebill1049** [reply]
one of my fav song :)
